packages/extension/gulpfile.ts

```diff
@@ -15,7 +16,11 @@ const manifest = () => {
 }
 
 const background = (cb) => {
-  exec('node ../../scripts/run-webpack.js').then(() => cb()).catch(cb)
```

Comment:
I thought `exec` had a second arg where you could pass

```js
exec('node ....', { stdout: ... }
```

But now I realize that is `execa`. Another option would be to grab the stdout from the callback:

```js
exec('node ...', (error, stdout, stderr) => {
  // stuff
})
```

It might be more idiomatic to do something like this, and do a `if (err)` check?

Either way is fine with me, I'll leave the final decision to you.

Reply:
I like the current implementation as it preserves the ansii highlighting, whereas console logging the `stdout` does not.